src/styles/Modal.scss

```diff
@@ -1,12 +1,11 @@
 .str-chat__modal {
-  background: var(--black);
+  background: var(--modal-overlay-color);
```

Comment:
@DanC5 this affects the React SDK as well

The problem was that not only the modal layout was transparent, but the modal itself (before and after screenshots attached)
<img width="1784" alt="Screenshot 2021-11-15 at 17 11 14" src="https://user-images.githubusercontent.com/6690098/141818994-e92c5312-92a5-48eb-822f-d34f16e7c2c0.png">

<img width="1271" alt="Screenshot 2021-11-15 at 17 21 09" src="https://user-images.githubusercontent.com/6690098/141819005-d3099bec-687c-4df2-b20b-efb1ee741dda.png">

Reply:
@DanC5 for some reason I couldn't add you as a reviewer, but please review this MR